src/renderers/dom/stack/client/ReactDOMComponent.js

```diff
@@ -583,6 +584,20 @@ ReactDOMComponent.Mixin = {
         );
         didWarnShadyDOM = true;
       }
+
+      if (__DEV__) {
+        if (this._namespaceURI === DOMNamespaces.html) {
+          warning(
+            !(document.createElement(this._tag) instanceof window.HTMLUnknownElement) ||
```

Comment:
Can we just use the element that was already created [above](https://github.com/facebook/react/blob/master/src/renderers/dom/stack/client/ReactDOMComponent.js#L569) (stored in the `el` variable)?

If there's some reason we need to create another element here, we should use `ownerDocument` instead of `document` to create the element.